guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -26,7 +26,34 @@ You cannot use ISS to synchronize content from {ProjectServer} to {SmartProxySer
 {SmartProxyServer} supports synchronization natively.
 For more information, see {PlanningDocURL}chap-Documentation-Architecture_Guide-Capsule_Server_Overview[{SmartProxyServer} Overview] in _Planning for {ProjectNameX}_.
 
-==== Air gapped
+=== Configuration Definitions
+There are different configurations or ways of using ISS to synchrone content between 2 {ProjectServer}s.
+
+.Connected
+The 2 {ProjectServer}s are connected and are able to communicate with each other.
+
+ifndef::satellite[]
+image::Connected.png[title="The {Project} ISS Connected use case", alt="The {Project} ISS Connected use case"]
+endif::[]
+
+.Non Air-gapped Disconnected
+* The 2 {ProjectServer}s are connected to each other.
+* However the Disconnected {ProjectServer} is not connected to the Internet. It is completely isolated from the outside world but is able communicate with a connected {ProjectServer}.
+
+ifndef::satellite[]
+image::Non-Airgapped-Disconnected.png[title="The {Project} ISS Non Airgapped Disconnected use case", alt="The {Project} ISS Airgapped Disconnected use case"]
+endif::[]
+
+
+.Air-gapped Disconnected
+* The 2 {ProjectServer}s are completely disconnected from each other.
+
+ifndef::satellite[]
+image::Airgapped-Disconnected.png[title="The {Project} ISS Airgapped use case", alt="The {Project} ISS Airgapped use case"]
+endif::[]
+
+
+==== Proxied Content
```

Comment:
I don't think that this title reflects the section 
```suggestion
=== Importing Content from a Connected to a Disconnected {ProjectServer}
```
There's also no need to bury it so deep. This way it will appear in the TOC also. 
Something like this would be better.

Reply:
This definitely needs work :(

So idea is Proxied Content and Managed content are applicable to both Connected and Disconnected. The mechanism for connected will change in the future, but for now Connected/Disconnected/Airgapped all use the import/export workflow. So this section trying to highlight `I want the server connected to the CDN to purely act as a glorified Content Depot (I am exporting only library here.)` vs `I want the server connected to the CDN to not only be a Content Depot but also Manage Content via Content Views/Lifecycle Environments (I am exporting both library and versions )`

Reply:
I think that we can add what you've said here under the titles for each of the procedures. 
They will be good short descriptions. 
Proxied content means nothing in and of itself as a title, so I think my suggestion is better and then we can really define the scenarios this applies to underneath. As the mechanisms change, we can add another and define that scenario clearly also.

Reply:
How about 

```suggestion
==== CDN connected {ProjectServer} to Download Content
```

or 
```suggestion
==== {ProjectServer} connected to the CDN as a Content Store
```

Reply:
Move over, we've a new tech writer in the house ;-) 

LGTM @parthaa thanks!